src/Team-Services-Bot.Api.UnitTests/Services/VstsServiceTest.cs

```diff
@@ -493,6 +488,42 @@ public class VstsServiceTest
         }
 
         [TestMethod]
+        [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Test method shouldn't be a property. Test method name corresponds to method under test.")]
+        public async Task GetReleasesTest()
```

Comment:
The test name doesn't match method under test. Should be GetRleaseDefinitoinsAsyncTest.